src/Http/Route.php

```diff
@@ -115,83 +155,102 @@ public function getAttributes(): array
      * Factories new instance of the route
      * that will be matched against get request.
      *
-     * @param string $expression
+     * @param string $path
+     * @param string $name
      * @return Route
      */
-    public static function get(string $expression): Route
+    public static function get(string $path, string $name = null): Route
     {
-        return new self($expression, Request::METHOD_GET);
+        return new self($path, [Request::METHOD_GET], $name ?? self::generateNameFromPath($path));
```

Comment:
`?? self::generateNameFromPath($path))` is not needed in those helper factory methods - constructor takes the `$name` as optional and does the same.